lib/utils/fs_test.go

```diff
@@ -38,10 +38,29 @@ func TestOpenFileLinks(t *testing.T) {
 	// broken-s -> nonexistent
 	// hardfile
 	// hardfile-h -> hardfile
-	rootDir := t.TempDir()
+
+	// macOS is special cased here since t.TempDir() returns a path to a protected directory that doesn't allow symlinks.
+	var rootDir string
+	var err error
+	switch runtime.GOOS {
+	case "darwin":
+		rootDir, err = os.MkdirTemp("/private/tmp", "teleport-test-*")
+		if err != nil {
```

Comment:
No need for the if statement anymore.

Reply:
Ah sorry, dumb mistake, it's late. Time to head out. Fixed